cli/stubs/secure.valet.conf

```diff
@@ -48,46 +48,3 @@ server {
         deny all;
     }
 }
-
-server {
-    listen 127.0.0.1:60;
```

Comment:
@mikaelpopowicz I think this port 60 part needs to stay in this file, which essentially then also removes this file from the PR.
(The port 60 stuff is used by ngrok when running the valet share command.)

Reply:
Yes, it was a mistake.